City-of-Helsinki/helsinki-paatokset
# PP-204: Ahjo cleanup and small fixes

**To test**
* Checkout branch, run `drush cim -y;drush cr;`
* Go to the modules page and make sure the "paatokset_ahjo" module is disabled: https://helsinki-paatokset.docker.so/fi/admin/modules
* If you don't have Ahjo date on your local environment, run: `drush ap:fs -v;drush mim ahjo_meetings:latest;drush mim ahjo_cases:latest;drush mim ahjo_decisions:latest;drush mim ahjo_decisionmakers:all;drush mim ahjo_trustees:council;drush ap:ud --localdata`
* Some templates and blocks were moved from the noew disabled "paatokset_ahjo" module to "paatokset_ahjo_api", so we should double check that those still work:
  * Modify a few kaupunginvaltuusto meetings to be in the future: https://helsinki-paatokset.docker.so/fi/admin/content/meetings?title=valtuusto&field_meeting_id_value=&field_meeting_minutes_published_value=All&field_meeting_agenda_published_value=All
    * These should now  appear on the valtuusto page: https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto
  * Check that the meeting calendar page works: https://helsinki-paatokset.docker.so/fi/kokouskalenteri
    * If you don't have this page locally, create a new landing page with /kokouskalenteri as the alias 
* The breadcrumbs should now be truncated and forced to be on one line: https://helsinki-paatokset.docker.so/fi/asia/hel-2022-000289/2fa818b4-28aa-4ad4-b9e7-1dd46a8b0c19
* The decision dropdown date should now reflect the meeting and not the decision issued date:
  * https://helsinki-paatokset.docker.so/fi/asia/hel-2021-013782/8a902780-0424-45d2-93ab-3b6c5acb62ed
  * https://helsinki-paatokset.docker.so/fi/asia/hel-2021-014145/5c3824ab-7e18-417a-82cf-b3b456d3c57b 
  * The date in the dropdown should be different than the one in the "appeal process" accordion element
* Check that the phone numbers in the "Ask for more info" section start with a "09" and not "310", for example:
  * https://helsinki-paatokset.docker.so/fi/asia/hel-2020-009550/b1f37575-d5ec-43a6-8dad-37559d681dc0
  * Check a few other decisions (including ones in Swedish) to see if the string replacement works uniformly
* The official decisions list should now section data (§) included in the titles: https://helsinki-paatokset.docker.so/fi/paattajat/pormestari/paatokset
